src/library/scala/Symbol.scala

```diff
@@ -14,12 +14,6 @@ package scala
 
 /** This class provides a simple way to get unique objects for equal strings.
  *  Since symbols are interned, they can be compared using reference equality.
- *  Instances of `Symbol` can be created easily with Scala's built-in quote
- *  mechanism.
```

Comment:
recommend `sym` instead...?

Reply:
nbd, but s/short-hand syntax/string interpolator/  I think

Reply:
Works for me